hamuli/Teamwork
# #168414346: user profile

#### What does this PR do?
- this PR allow user to edit , delete , comment and share posts.
#### Description of Task to be completed?
- delete : user can delete his own  post
- edit : user can edit his own post
- comment on his own post and reply to a comment of someone else.
- post a new post;
#### How should this be manually tested?
- when user delete and see the post be deleted.
- when user edit and see the change on his on his post
- when user comment , reply on his own and other posts.
- when user post and see his post be commented by others. 
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/168412767
#### Screenshots
